deps/verifier/src/cca/mod.rs

```diff
@@ -227,27 +210,28 @@ impl Verifier for CCA {
 ///    }
 /// }
 /// NOTE: each of the value are base64 encoded hex value.
-fn parse_cca_evidence(evidence_map: &BTreeMap<String, RawValue>) -> Result<Evidence> {
-    let mut evidence = Evidence::default();
-    let platfrom = evidence_map
-        .get("platform")
-        .context("get platform evidence from the cca evidence map")?;
-    let output = serde_json::to_string(platfrom)?;
+fn parse_cca_evidence(
```

Comment:
This comment doesn't relate directly to this line, but it's important to know that Trustee now basically requires that verifiers return a `report_data` claim (or a secure channel won't be established) and encourages verifiers to return an `init_data` claim if they check the initdata (which you do). Adding these values to the claims signals that the verifier has checked these properties.

Reply:
Apologies. To make progress on this, I need to understand two things:

1. how to populate `report_data` and `init_data` (with the successfully matched values?), and
2. where to put those claims (in the `TeeEvidenceParsedClaim`?)

Thanks for your help!

Reply:
And a third thing: what is the expected encoding? Base64 (padded?), base64url (padded?), Base16, other?

Reply:
See my shot in the dark [here](https://github.com/confidential-containers/trustee/compare/e6c14169f06e9fe76abf20c52da379c3d3b1524e..e3a950565bf64888de48446eb83508b33374077d), which produces:

```json
“[...]"
"init_data": "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA==",
"init_data_claims": null,
"report_data": "LJz1gp5O3CVbqAWUDeJWiKVZJJYva4zqStzj6JowYnXOTHZfv7UL1fKiYekjx9j9AAAAAAAAAAAAAAAAAAAAAA==",
```

Reply:
Hey @thomas-fossati . About these Qs
1. How to populate init/report data & 2.Where to place
This [commit](https://github.com/confidential-containers/trustee/compare/e6c14169f06e9fe76abf20c52da379c3d3b1524e..e3a950565bf64888de48446eb83508b33374077d) looks good!

3. Initdata/report data field Encoding.
I think you can use hex which all other platforms s.t. SGX/TDX/SNP/SE/vTPM are using.